### DISCUSSION TITLE
Security Onion with Wazuh.

### DISCUSSION THREAD

**Discussion by nguyenhuukhoi** [Security-Onion-Solutions/securityonion#7287] [Unsupported Versions] [answered]
Hello everyone.
I am very interested with SO. It is  a great tool. However, I want to integrate it with my Wazuh. Is there anyway to do it. Many thanks for your helps.

**Comment by dougburks** [comment, marked as answer]
Security Onion already includes Wazuh:
https://docs.securityonion.net/en/2.3/wazuh.html

You may be able to integrate with a separate Wazuh installation, but we don't recommend or support it.

**Reply by nguyenhuukhoi** [reply]
Yep. I hope we will support it. Bc I see so-wazuh too old than current wazuh version.
